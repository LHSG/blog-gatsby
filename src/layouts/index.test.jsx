import React from 'react';
import { shallow } from 'enzyme';

import IndexTemplate from '.';

test('rendering', () => {
  const props = {
    location: { pathname: 'pathname' },
    children: () => 'children',
  };
  const wrapper = shallow(<IndexTemplate {...props} />);
  expect(wrapper).toMatchSnapshot();
});
