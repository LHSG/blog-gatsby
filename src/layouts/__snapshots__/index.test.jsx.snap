// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering 1`] = `
<Layout
  path="pathname"
/>
`;
